test/test_yaml.py

```diff
@@ -189,3 +191,14 @@ def test_update_fn(self):
         data = to_yaml(data)
         card.update(data)
         self.assertEqual(card.formatted_name, fn)
+
+    def test_parse_field_with_colon(self):
+        card = create_test_card()
+        fn = 'me myself and i'
+        data = {'Formatted name': fn}
+        data = to_yaml(data)
+        print(khard.helpers.convert_to_yaml("Note", "foo: bar", 0, 5, True))
+        data += "\n".join(khard.helpers.convert_to_yaml("Note", "foo: bar", 0,
+                                                        5, True))
+        card.update(data)
```

Comment:
What we actually should test for are two things (in two tests):
1. that convert_to_yaml handles strings with colons special
2. more important that a round trip of a vcard to yaml and back is idempotent
   1. in general
   2. specially for values with colons

Reply:
So 1 test as above, but with an additional non-colon value, and another test for (1)?

Reply:
I was deliberately avoiding making the test specific to this behaviour (re point 1), but I can do that.

Reply:
I think that should all be covered now.